clang/test/Driver/openmp-offload.c

```diff
@@ -661,12 +661,13 @@
 /// ###########################################################################
 
 /// test behaviors of -foffload-static-lib=<lib>
-// RUN: touch %t.a
+// RUN: echo "void foo(void) {}" > %t1.cpp
```

Comment:
Just to make sure I understand: is it that the `%t1_bundle.o` object needs to be a "true" one now that `clang-offload-bundler` receives an archive input and would fail on a dummy object with a `File too small` error?

Reply:
What I'm addressing here is the fact that the usage of `-foffload-static-lib` doesn't guarantee that the archive is recognized as an actual offload archive.  Making it a true offload archive will properly trigger the behavior.  We were able to get away with it with the partial-link path as everything was thrown into the partial-link step.  I guess you can say its a precursor to the removing of -foffload-static-lib support.